.github/workflows/ghcr-deploy.yml

```diff
@@ -3,11 +3,13 @@ name: Create and publish a Docker image
 
 on:
     push:
-        # Push events to `release/*` branches will trigger the workflow
-        # for example, `release/1.0.0`, `release/1.0.1`, `release/2.0.0`
-        branches: ["release/*"]
+        branches:
+            - main
+        tags:
+            - v*
+    schedule:
+        - cron: "0 0 * * *"
```

Comment:
note to self to discuss this tomorrow during the meeting, not sure if nightly makes sense for our use cases, and might hit our org usage quota